straxen/plugins/peaklet_processing.py

```diff
@@ -543,13 +543,21 @@ def compute(self, records_he, start, end):
 
 @export
 @strax.takes_config(
-    strax.Option('s1_max_rise_time', default=110, infer_type=False,
-                 help="Maximum S1 rise time for < 100 PE [ns]"),
-    strax.Option('s1_max_rise_time_post100', default=200, infer_type=False,
+    strax.Option('s1_risetime_area_parameters', default=(50, 80, 12), type=(list, tuple),
+                 help="norm, const, tau in the empirical boundary in the risetime-area plot"),
+    strax.Option('s1_risetime_aft_parameters', default=(-1, 2.6), type=(list, tuple),
+                 help=("Slope and offset in exponential of emperical boundary in the rise time-AFT "
+                      "plot. Specified as (slope, offset)")),
+    strax.Option('s1_flatten_threshold_aft', default=(0.7, 100), type=(tuple, list),
+                 help=("Threshold for AFT, above which we use a flatted boundary for rise time" 
+                       "Specified values: (AFT boundary, constant rise time).")),
+    strax.Option('n_top_pmts', default=straxen.n_top_pmts, type=int,
+                 help="Number of top PMTs"),
+    strax.Option('s1_max_rise_time_post100', default=200, type=(int, float),
```

Comment:
This current 200 ns cut seems bit loose to me. Do you (or @zhut19 / @WenzDaniel ) have any suggestions?

Reply:
I personally think we can just set it as the constant (B=80ns) in the <100 PE classification boundary, as we know the fluctuation is less for large peaks. But it depends on whether we want to keep the gas S1s.

Reply:
Thanks, I agree that we probably need other inputs to optimize this. So it's fine as it is now at the momement